tests/Microsoft.Bot.Builder.Tests/MemoryConnectorClient.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.using System.Security.Claims;
+
+using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Security.Claims;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Microsoft.Rest;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Moq;
+using Moq.Protected;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Bot.Builder.Tests
+{
+    public class MemoryConnectorClient : IConnectorClient
```

Comment:
Why is this here?

Reply:
I use it in my tests...what do you mean? 

It's used to capture the calls that would be made to the ConnectorClient so I can run through tests Functionally without having to make any calls.

Reply:
Tests!